mne/viz/ica.py

```diff
@@ -373,12 +433,21 @@ def set_title_and_labels(ax, title, xlab, ylab):
         if idx > 0:
             fig, axes = _create_properties_layout(figsize=figsize)
 
+        # we reconstruct an epoch_variance with 0 where indexes where dropped
+        epoch_var = np.var(ica_data[idx], axis=1)
+        drop_var = np.var(dropped_src[idx], axis=1)
+        epoch_var = np.insert(epoch_var,
+                              (dropped_indices -
```

Comment:
you have a bunch of intermediate variables and here you computed inline. Why? 

If you have already gone with intermediate vars I would do:
```py
epoch_var = np.var(ica_data[idx], axis=1)
drop_var = np.var(dropped_src[idx], axis=1)[dropped_indices]
nice_var_name = (dropped_indices - np.arange(len(dropped_indices))).astype(int)
epoch_var = np.insert(epoch_var, nice_var_name, drop_var, axis=0)
```
And let the var names document what am I inserting, or I would put the computations
in the call (skip the variable name choosing all together) and let the parameters
of the insert document what I'm doing. Like this:

```py
epoch_var = insert(arr=np.var(ica_data[idx], axis=1),
                   obj=np.var(dropped_src[idx], axis=1)[dropped_indices],
                   values=(dropped_indices -
                           np.arange(len(dropped_indices))).astype(int),
                   axis=0)
```